docs: add mermaid configuration to content routes (#723)
analogjs/analog: apps/docs-app/docs/features/routing/content.md

}
```

### Enabling support for mermaid

Analog's markdown component supports [Mermaid](https://mermaid.js.org/). To enable support by the `MarkdownComponent` define a dynamic import for `loadMermaid` in `withMarkdownRenderer()`.

```ts
withMarkdownRenderer({
  loadMermaid: () => import('mermaid'),
});
```

> Once it is enabled, mermaid blocks will be transformed by mermaid in svgs.

Reviewer: ```suggestion
After it is enabled, mermaid blocks are transformed by mermaid into svgs.
```
